u2f-api-polyfill.js

```diff
@@ -11,7 +11,8 @@
 'use strict';
 
 (function (){
-  if ('u2f' in window || !('chrome' in window)) {
+  var isChrome = 'chrome' in window && !window.navigator.userAgent.includes('Edge');
```

Comment:
This is fine. I'm not aware of any APIs that we could feature-detect that are proprietary to Edge and won't be ever coming to other browsers.